M2/Macaulay2/packages/LocalRings.m2

```diff
@@ -40,8 +40,8 @@ newPackage(
     AuxiliaryFiles => true
     )
 
-importFrom_Core { "ContainmentHooks", "ReduceHooks", "printerr",
-    "raw", "rawLiftLocalMatrix", "rawMatrixRemake2", "rawSource" }
+importFrom_Core { "ContainmentHooks", "ReduceHooks", "raw",
+    "rawLiftLocalMatrix", "rawMatrixRemake2", "rawSource" }
```

Comment:
If you make a new commit, I'd prefer if you also move `"raw"` back on the second line, with everything else that starts with `"raw"`.

Reply:
Fair enough -- will do.